src/controllers/player.coffee

```diff
@@ -512,11 +600,45 @@ angular.module 'matching'
 				if matchesPerPage[page] < pairsPerPage[page]
 					if matchesPerPage[$scope.currentPage] == pairsPerPage[$scope.currentPage]
 						$scope.unfinishedPagesBefore = true
+
 		unless $scope.currentPage == $scope.pages.length - 1
 			for page in [$scope.currentPage..pairsPerPage.length]
 				if matchesPerPage[page] < pairsPerPage[page]
 					if matchesPerPage[$scope.currentPage] == pairsPerPage[$scope.currentPage]
 						$scope.unfinishedPagesAfter = true
 
+	$scope.unfocused = () ->
+		$scope.helpVisible = !$scope.helpVisible
+		helpButton = document.getElementById("help-button");
+		gameBoard = document.getElementById("gameboard");
+		panelButtons = document.getElementById("page-selector");
+		finishButton = document.getElementById("finish-button");
+
+
+		panelButtons.inert = true;
+		helpButton.inert = true;
+		gameBoard.inert = true;
+		finishButton.inert = true;
+
+		helpModal.inert = false;
+
+	$scope.focused = () ->
+
+		$scope.helpVisible = false
+		helpButton = document.getElementById("help-button");
+		gameBoard = document.getElementById("gameboard");
+		panelButtons = document.getElementById("page-selector");
+		finishButton = document.getElementById("finish-button");
+
+
+		panelButtons.inert = false;
```

Comment:
See my previous comment about `inert` attributes. These assignments will need to be updated to use `removeAttribute` instead, on lines 634-640.